components/tidb_query_vec_executors/src/runner.rs

```diff
@@ -154,6 +159,7 @@ pub fn build_executors<S: Storage + 'static>(
                     ranges,
                     primary_column_ids,
                     descriptor.get_desc(),
+                    is_streaming,
```

Comment:
Could you explain how `is_streaming` associated with `is_scanned_range_aware` via comments?

```rust
// the reason
let is_scanned_range_aware = is_streaming;
```